src/main/kotlin/com/sourcegraph/cody/chat/Chat.kt

```diff
@@ -92,9 +92,9 @@ class Chat {
                       "(Already upgraded to Pro? Restart your IDE for changes to take effect)"
                 }
                 else -> {
-                  "<b>Request failed:</b> You've used all${rateLimitError.quotaString()} chat messages and commands." +
```

Comment:
Out of curiosity why we are not using text bundle everywhere but instead copy paste the same message in different places?

Reply:
I think we should use the bundle everywhere. I initialized that during the rough time around GA for some specific purpose - the bundle makes things cleaner and more readable imho.